### PR TITLE
Correct `sort` data when multiple files upload

### LINKED ISSUE #4303: [Panel] Sorting files messed up when uploading multiple files
## Description
Uploading multiple files at the same time makes them have all the same sorting number. 

**Expected behavior**  
Files should have sequential sorting numbers. 

## To reproduce
1. Create a files section blueprint:
```yml
    sections:
      images:
        type: files
        limit: 4
        text: "fig. {{ file.sort }}"
``` 
2. Upload 5 files (in the same drag & drop action)
3. See that all uploaded files have the same sorting number. 
4. Drag the first file to the second position
5. Notice how the dragged file is (probably) not in the second position

## Your setup
**Kirby Version**  
3.6.6

**Your system (please complete the following information)**
 - Device: Desktop
 - OS: Windows
 - Browser: Chrome
 - Version: 101

**Additional context**  
related? https://github.com/getkirby/kirby/issues/2886

### PR DESCRIPTION
**Pull request by afbora**
## This PR …

In multi-file upload, I fixed the `sort` data on Vue side, because a single request is made on PHP side and uploading files looped on Vue side. I'm not sure it's the right solution.

### Fixes

- Sorting files messed up when uploading multiple files #4303

### Breaking changes

None

## Ready?

- [ ] Unit tests for fixed bug/feature
- [x] In-code documentation (wherever needed)
- [x] Tests and checks all pass

### For review team

- [x] Add changes to release notes draft in Notion
- [x] Add to [website docs release checklist](https://github.com/getkirby/getkirby.com/pulls) (if needed)

### CODE REVIEW

File: panel/src/components/Forms/Upload.vue [hunk 1]
```diff
@@ -158,6 +158,12 @@ export default {
 
       this.total = this.files.length;
       this.files.forEach((file) => {
+        // if there is sort data, increment it in the loop
+        // the first `sort` data is the sort of the last file uploaded on the app
+        if (this.options?.attributes?.sort) {
+          this.options.attributes.sort++;
```
Comment (distantnative) [inline panel/src/components/Forms/Upload.vue]: Are we using `this.options.attributes.sort` anywhere else here? Otherwise this seems to be fine

Comment (afbora) [reply panel/src/components/Forms/Upload.vue]: As far as I know and look, it is not used elsewhere.

**Comment by distantnative** [comment]
Looks good to me from reading but I should try it out a few times/ways first and then approve.

**Comment by distantnative** [comment]
Something is broken: If I follow the reproducing instructions from https://github.com/getkirby/kirby/issues/4303 with this branch, I get
Screen Shot 2022-05-25 at 19 42 12

Same, meta file will say `Sort: NaN`

**Comment by afbora** [comment]
@distantnative Thanks for testing. I've changed the process. Is it working now?

**Comment by distantnative** [comment]
Yes, it does

**Review by distantnative — APPROVED** [review]

**Pull request merged**